tests/conftest.py

```diff
@@ -24,13 +24,15 @@ def threaded_dask():
     dask.config.set(scheduler="threads")
 
 @pytest.fixture(scope='function')
-def storage_config(tdb_filepath, bounds, resolution, crs, attrs, metrics):
+def storage_config(tdb_filepath, bounds, resolution, htthreshold, coverthreshold, crs, attrs, metrics):
     log = Log(20)
     yield StorageConfig(tdb_dir = tdb_filepath,
                         log = log,
                         crs = crs,
                         root = bounds,
                         resolution = resolution,
+                        htthreshold = htthreshold,
```

Comment:
Are these parameters a definition of the Metric or of the Storage? I get that it is a per-db setting but I wonder if we should store the value of the thresholds in the metric (which is stored in the db) instead of at the top-level StorageConfig.

Additionally, this makes it clear that we need another level of `ConfigConfig` 😄 to consolidate this kind of stuff. If we were to add another couple of parameters, we're getting really long in the tooth.

Reply:
For ultimate flexibility, store threshold with metric but I think most use cases would have the same threshold values across all metrics. Still not sure how we deal with applying the height threshold when computing metrics for other dimensions (non height). It gets pretty confusion if we have a height threshold stored with a metric value computed using a different dimension.